utils/notification_service.py

```diff
@@ -920,6 +922,8 @@ def prepare_reports(title, header, reports, to_truncate=True):
 
 if __name__ == "__main__":
     SLACK_REPORT_CHANNEL_ID = os.environ["SLACK_REPORT_CHANNEL"]
+    REPORT_REPO_ID = os.environ.get("REPORT_REPO_ID", "hf-internal-testing/transformers_daily_ci")
```

Comment:
hmm, IIRC, `REPORT_REPO_ID` will exist as you define it in `slack-report.yml`, despite it could be empty string if not from AMD (i.e. for Nvidia CI). And we need to change the condition here. 

(just like what you did one line below for `UPLOAD_REPORT_SUMMARY`)